homeassistant/components/airzone/sensor.py

```diff
@@ -76,11 +74,11 @@ def __init__(
         description: SensorEntityDescription,
         entry: ConfigEntry,
         system_zone_id: str,
-        zone_name: str,
+        zone_data: dict,
```

Comment:
Please make the dict more specific

Reply:
Fixed in 66bd22912bbcf6577569050a66093a5dd4394fab